db/schema.rb

```diff
@@ -97,6 +97,11 @@
     t.bigint "authority_id"
   end
 
+  create_table "suggest_a_registers", force: :cascade do |t|
```

Comment:
This contradicts the commented out migration

Reply:
I've removed both.